MovieApp/MovieApp/App/Coordinator/AppRouter.swift

```diff
@@ -44,4 +44,34 @@ extension AppRouter: AppRouterProtocol {
         window?.makeKeyAndVisible()
     }
     
+    func showTabBarView() {
+        let tabBarController = createTabBarController()
+        
+        navigationController.setViewControllers([tabBarController], animated: true)
+    }
```

Comment:
Since the TabBar will always be the initial screen, you can remove this function and add this logic to `setInitialScreen`.

Reply:
Fixed [here](https://github.com/lihter/Movie-App/pull/24/files#diff-b8e8fb675a7084dca12ae6abb54b13ee1d7293192f1e1e75308b4dca15ea5465R30)